core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java

```diff
@@ -16,5 +23,60 @@ public void testResolve_knownTokens() {
 
         assertThat(Long.parseLong(uniqueId) > 1, is(true));
 
+    }	
+    
+    @Test
+    public void testFixedRandomGenerator_success() {
+    	IntStream.rangeClosed(1, 19).forEach(i->{
+    		assertTrue(FixedRandomGenerator.getGenerator(i).generateRandomNumber().length() == i);
+    	});
+    }
+    
+    @Test
+    public void testFixedRandomGenerator_failure_min() {
+    	exceptionRule.expect(RuntimeException.class);
+    	FixedRandomGenerator.getGenerator(0).generateRandomNumber();
+    }
+    
+    @Test
+    public void testFixedRandomGenerator_failure_max() {
+    	exceptionRule.expect(RuntimeException.class);
+    	FixedRandomGenerator.getGenerator(20).generateRandomNumber();
     }
-}
\ No newline at end of file
+    
+    @Test
+    public void testFixedRandomTokenReplace() {
+    	 String clientId = "zerocode-clientid_${RANDOM.NUMBER:10}";
+         String uniqueClientId = resolveKnownTokens(clientId);
+         String uniqueId = uniqueClientId.substring("zerocode-clientid_".length());
+         assertTrue(uniqueId.length() == 10);
+    }
+    
+    @Test
+    public void testFixedRandomUniqueness(){
+    	String result = resolveKnownTokens("${RANDOM.NUMBER:12},${RANDOM.NUMBER:12}");
+    	String[] split = result.split(",");
+    	assertTrue(split[0] != split[1]);
+    }
+    
+    
+}
+
```

Comment:
Minor one - These empty line basically showing up here...
Can you please delete them